src/material/core/tokens/_m3-system.scss

```diff
@@ -84,35 +152,32 @@ $_system-level-prefix: sys;
   $shadow-color: map.get(
       $theme, _mat-theming-internals-do-not-access, color-tokens, (mdc, theme), shadow);
 
-  @for $level from 0 through 24 {
-    $value: elevation.get-box-shadow($level, $shadow-color);
-    --#{$prefix}-elevation-shadow-level-#{$level}: #{$value};
-  }
-
   @each $name, $value in definitions.md-sys-elevation-values() {
     $level: map.get($overrides, $name) or $value;
     $value: elevation.get-box-shadow($level, $shadow-color);
-    --#{$prefix}-#{$name}: #{$value};
+    & {
+      --#{$prefix}-#{$name}: #{$value};
+    }
   }
 }
 
-@mixin system-level-shape($theme, $overrides: (), $prefix: $_system-level-prefix) {
+@mixin system-level-shape($theme: (), $overrides: (), $prefix: $_system-level-prefix) {
```

Comment:
I feel like requiring a `theme` config makes it seem like it does anything here, but its ignored in the mixin since our themes dont actually allow this to be configured. 

That said, Im not strongly opinionated about making this optional - we can leave it the way it is

Reply:
Ah I see, the theme only has an effect in the color and typography mixins. In that case it's fine by me.